src/spend.rs

```diff
@@ -575,6 +576,8 @@ pub struct CreateSpendRes {
 /// * `main_descriptor`: the multipath Liana descriptor, used to derive the addresses of the
 /// candidate coins.
 /// * `secp`: necessary to derive data from the descriptor.
+/// * `use_primary_path`: whether the primary spending path should be used when estimating
+/// satisfaction size.
```

Comment:
Technically we already have this information from the nSequence to use. But maybe it's clearer by using a boolean, i don't feel strongly.

Reply:
OK great, I think it would be good to avoid adding a parameter in that case. I'll try changing it to get the info from nSequence instead.

Reply:
I've now removed the parameter and instead check the candidates' nSequence values.